<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>宋体</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>AHRS配置软件</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="1">
     <widget class="QWidget" name="widget_left" native="true">
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="12" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>原始包间隔:(ms)</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>陀螺z轴:(deg/s)</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>陀螺y轴:(deg/s)</string>
         </property>
        </widget>
       </item>
       <item row="16" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>四元数w</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>陀螺x轴:(deg/s)</string>
         </property>
        </widget>
       </item>
       <item row="17" column="0">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>四元数x</string>
         </property>
        </widget>
       </item>
       <item row="15" column="0">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>zyx欧拉角z:(deg)</string>
         </property>
        </widget>
       </item>
       <item row="18" column="0">
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>四元数y</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>磁罗盘x轴:(G)</string>
         </property>
        </widget>
       </item>
       <item row="14" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>zyx欧拉角y:(deg)</string>
         </property>
        </widget>
       </item>
       <item row="13" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>zyx欧拉角x:(deg)</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="cb_uart"/>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="bt_open">
         <property name="text">
          <string>打开串口</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>加计y轴:(g)</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>磁罗盘y轴:(G)</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>磁罗盘z轴:(G)</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>加计z轴:(g)</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>加计x轴:(g)</string>
         </property>
        </widget>
       </item>
       <item row="19" column="0">
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>四元数z</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="lb_g_x">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="lb_g_y">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLabel" name="lb_g_z">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLabel" name="lb_a_x">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLabel" name="lb_a_y">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QLabel" name="lb_m_x">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QLabel" name="lb_m_z">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QLabel" name="lb_a_z">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QLabel" name="lb_m_y">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="16" column="1">
        <widget class="QLabel" name="lb_quat_w">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="13" column="1">
        <widget class="QLabel" name="lb_eu_x">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="12" column="1">
        <widget class="QLabel" name="lb_delta_time">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="15" column="1">
        <widget class="QLabel" name="lb_eu_z">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="14" column="1">
        <widget class="QLabel" name="lb_eu_y">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="19" column="1">
        <widget class="QLabel" name="lb_quat_z">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="18" column="1">
        <widget class="QLabel" name="lb_quat_y">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="17" column="1">
        <widget class="QLabel" name="lb_quat_x">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_31">
         <property name="text">
          <string>波特率：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="cb_baud"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="2">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="0">
       <widget class="QWidget" name="widget_para" native="true">
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="0" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>磁力计比例x</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_20">
           <property name="text">
            <string>磁力计比例y</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QLineEdit" name="le_m_0_z"/>
         </item>
         <item row="1" column="3">
          <widget class="QLineEdit" name="le_m_0_y"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_27">
           <property name="text">
            <string>串口波特率</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QLabel" name="label_26">
           <property name="text">
            <string>姿态包分频</string>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QLineEdit" name="le_fre_2"/>
         </item>
         <item row="5" column="1">
          <widget class="QComboBox" name="cb_set_baud"/>
         </item>
         <item row="6" column="3">
          <widget class="QPushButton" name="bt_write_cfg">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;将配置信息写入传感器，但不保存&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;若需保存，需按“保存配置”按钮&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string>写入配置</string>
           </property>
          </widget>
         </item>
         <item row="6" column="4">
          <widget class="QPushButton" name="bt_dft_cfg">
           <property name="text">
            <string>恢复默认值</string>
           </property>
          </widget>
         </item>
         <item row="6" column="5">
          <widget class="QPushButton" name="bt_save_cfg">
           <property name="text">
            <string>保存配置</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_22">
           <property name="text">
            <string>磁力计0偏x</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="le_m_kx"/>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="le_m_ky"/>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="le_m_kz"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_21">
           <property name="text">
            <string>磁力计比例z</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4" rowspan="4" colspan="2">
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>非保存配置</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_8">
            <item row="2" column="0">
             <widget class="QLabel" name="label_30">
              <property name="text">
               <string>陀螺0偏z</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_28">
              <property name="text">
               <string>陀螺0偏x</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_29">
              <property name="text">
               <string>陀螺0偏y</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="le_g_0_x"/>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="le_g_0_y"/>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="le_g_0_z"/>
            </item>
           </layout>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_25">
           <property name="text">
            <string>原始包分频</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QPushButton" name="bt_read_cfg">
           <property name="text">
            <string>读取配置</string>
           </property>
          </widget>
         </item>
         <item row="6" column="2">
          <widget class="QPushButton" name="bt_cali_g">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;校准陀螺0位漂移，首先需使传感器完全静止&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string>校准陀螺</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="lb_state">
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 0, 0);</string>
           </property>
           <property name="text">
            <string>无连接</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="le_fre_1"/>
         </item>
         <item row="0" column="3">
          <widget class="QLineEdit" name="le_m_0_x"/>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="label_23">
           <property name="text">
            <string>磁力计0偏y</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="label_24">
           <property name="text">
            <string>磁力计0偏z</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_32">
           <property name="text">
            <string>曲线1：</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QComboBox" name="cb_curve0">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;选择曲线图中曲线1显示的数据&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="7" column="2">
          <widget class="QLabel" name="label_33">
           <property name="text">
            <string>曲线2：</string>
           </property>
          </widget>
         </item>
         <item row="7" column="3">
          <widget class="QComboBox" name="cb_curve1">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;选择曲线图中曲线2显示的数据&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="7" column="5">
          <widget class="QComboBox" name="cb_curve2">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;选择曲线图中曲线3显示的数据&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="7" column="4">
          <widget class="QLabel" name="label_34">
           <property name="text">
            <string>曲线3：</string>
           </property>
          </widget>
         </item>
         <item row="5" column="5">
          <widget class="QPushButton" name="bt_cali_m">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;进入罗盘校准状态时，记录校准点。&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;图中以0,0点为中心，显示x-y坐标分布图&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;以2,0点为中心，显示x-z坐标分布图&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;移动传感器，使两个分布图呈现完整的圆形&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;再次点击校准罗盘按钮，结束校准，结果显示在系数框中&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton:checked{background-color:rgb(85, 170, 255); border-style: inset; }</string>
           </property>
           <property name="text">
            <string>校准罗盘</string>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QCheckBox" name="cbox_9axis">
           <property name="text">
            <string>9轴模式</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QWidget" name="widget_chart" native="true">
        <layout class="QGridLayout" name="gridLayout_6"/>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>bt_open</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>slot_open()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bt_write_cfg</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>slot_write_cfg()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bt_dft_cfg</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>slot_dft_cfg()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bt_save_cfg</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>slot_save_cfg()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bt_read_cfg</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>slot_read_cfg()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bt_cali_g</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>slot_cali_g()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bt_cali_m</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>slot_cali_m()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_curve0</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>slot_curve_sel_change()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>389</x>
     <y>223</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_curve1</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>slot_curve_sel_change()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>563</x>
     <y>223</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_curve2</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>slot_curve_sel_change()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>740</x>
     <y>223</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot_open()</slot>
  <slot>slot_write_cfg()</slot>
  <slot>slot_dft_cfg()</slot>
  <slot>slot_save_cfg()</slot>
  <slot>slot_read_cfg()</slot>
  <slot>slot_cali_g()</slot>
  <slot>slot_cali_m()</slot>
  <slot>slot_curve_sel_change()</slot>
 </slots>
</ui>
